distribution/tools/keystore-cli/src/test/java/org/elasticsearch/common/settings/ChangeKeyStorePasswordCommandTests.java

```diff
@@ -54,14 +55,15 @@ public void testChangeKeyStorePassword() throws Exception {
         createKeystore("theoldpassword");
         loadKeystore("theoldpassword");
         terminal.addSecretInput("theoldpassword");
-        terminal.addSecretInput("thepassword");
-        terminal.addSecretInput("thepassword");
+        terminal.addSecretInput("the-better-password");
+        terminal.addSecretInput("the-better-password");
         // Prompted thrice: Once for the existing and twice for the new password
         execute();
```

Comment:
Do we need to execute(randomFrom("-p", "--password")) here as well?

Reply:
No, it's calling the `ChangeKeyStorePasswordCommand` (`elasticsearch-keystore passwd`) which doesn 't have a "--password" option (because it always deals with passwords).